po/es.po

```diff
@@ -380,7 +380,7 @@ msgstr "Experimental"
 
 #: ui/preferences-window.ui:268
 msgid "Use PipeWire for audio output"
-msgstr "Usar PipeWire para la salida de audio"
+msgstr "Usar PipeWire"
```

Comment:
Since PipeWire can handle both video and audio, this was explaining that we use it for audio only. Why remove?

Reply:
Not resolved. Unresolving.